platinum-sw-register.html

```diff
@@ -376,8 +374,26 @@
     },
 
     attached: function() {
+      var isSupported = 'serviceWorker' in navigator;
+
+      if (!isSupported) {
+        this._setState('unsupported');
+        return;
+      }
+
       if (this.autoRegister) {
-        this.async(this.register);
+        if (document.readyState !== 'complete' &&
+            document.readyState !== 'interactive') {
+
+          document.onreadystatechange = function() {
```

Comment:
This is the 2.0 change, right? Wanna add a small comment as to why it's needed? ("need to figure out if children are upgraded" or some sorts)